plugins/UsersManager/angularjs/personal-settings/personal-settings.controller.js

```diff
@@ -34,8 +34,26 @@
             });
         }
 
+        this.regenerateTokenAuth = function () {
+
+            self.loading = true;
+
+            piwikApi.withTokenInUrl();
+            piwikApi.post({
+                module: 'UsersManager', action: 'regenerateTokenAuth', format: 'json'
```

Comment:
Here we could call the API directly as well maybe just like in sites manager? Also mentioned this in `Controller.php` so we could get rid of that controller method